### PR TITLE
Update Black Desert Online

### PR DESCRIPTION
**Pull request by LadySmith**
Game had 2fa since 2018.

### CODE REVIEW

File: _data/gaming.yml [hunk 1]
```diff
@@ -18,7 +18,9 @@ websites:
       img: bdo.png
       twitter: BDO_News
```
Comment (kmpoppe) [inline _data/gaming.yml]:
```suggestion
```

File: _data/gaming.yml [hunk 2]
```diff
@@ -18,7 +18,9 @@ websites:
...
       facebook: BlackDesertOnline
```
Comment (kmpoppe) [inline _data/gaming.yml]:
```suggestion
```

File: _data/gaming.yml (hunk 1 above)
Comment (kmpoppe) [reply _data/gaming.yml]: When `tfa: Yes` social media links to tell them to add 2FA are no longer needed.

File: _data/gaming.yml [hunk 3]
```diff
@@ -18,7 +18,9 @@ websites:
...
-      tfa: No
+      tfa: Yes
+      software: Yes
+      doc: https://blackdesert.zendesk.com/hc/en-us/articles/360000535909-2-Step-Verification-FAQ
```
Comment (kmpoppe) [inline _data/gaming.yml]:
```suggestion
      doc: https://blackdesert.zendesk.com/hc/en-us/articles/360000535909
```

Comment (kmpoppe) [reply _data/gaming.yml]: Zendesk links don't need anything after the article's ID to work correctly.

**Comment by kmpoppe** [comment]
Hello @LadySmith and thank you for your contribution!
I've suggested and described a few changes, I'd be glad if you would be so kind as to commit them to your PR.
Thank you 😃

**Comment by LadySmith** [comment]
Thanks for your hard work.

**Review by kmpoppe — APPROVED** [review]
Funnily enough, the documetation page says that there are no other programs than Google Authenticator that can be used, where e.g. 1password is clearly able to scan the TOTP codes 😉

**Comment by LadySmith** [comment]
I played that game in the past, used Authy for that 😄

**Review by dsoegijono — APPROVED** [review]

**Pull request merged**